intro.html

```diff
@@ -0,0 +1,6 @@
+<html>
+    <head>
+<TITLE>Hello from Github!</TITLE>
+    </head>
+    <body</body>
```

Comment:
Can you add some content in the body
Thanks lad

Reply:
Sure thing. Oversight on my part.

Reply:
I'm going to add some Rocket League-related content.